control-plane/src/index.ts

```diff
@@ -12,7 +12,7 @@ const app = fastify({
 
 const s = initServer();
 
-app.register(s.plugin(router.router));
+app.register(s.plugin(router.router), { jsonQuery: true});
```

Comment:
This will break the existing client which will also need to be updated to enable `jsonQuery`.